Dalamud/Plugin/Internal/PluginManager.cs

```diff
@@ -207,6 +212,46 @@ public static bool IsManifestVisible(RemotePluginManifest manifest)
     /// <inheritdoc/>
     public void Dispose()
     {
+        if (!this.InstalledPlugins.Any())
```

Comment:
Why do you only dispose this if there are any plugins? It's applied every time - not that it matters, just wanna understant the thinking behind it

Reply:
Same code exists also at the end of Dispose, but now that I think about it again, less duplicates would be more preferrable than less indents in this case. Will invert conditionals.